metricbeat/module/docker/memory/_meta/data.json

```diff
@@ -1,61 +1,94 @@
+            }
+        },
         "memory": {
             "fail": {
                 "count": 0
             },
-            "limit": 2095878144,
+            "limit": 67514433536,
             "rss": {
-                "pct": 0.0004025882909345325,
-                "total": 843776
+                "pct": 0,
+                "total": 0
             },
             "stats": {
-                "active_anon": 421888,
-                "active_file": 36864,
-                "cache": 86016,
-                "dirty": 0,
-                "hierarchical_memory_limit": 9223372036854771712,
-                "hierarchical_memsw_limit": 9223372036854771712,
-                "inactive_anon": 421888,
-                "inactive_file": 49152,
-                "mapped_file": 53248,
-                "pgfault": 1587,
-                "pgmajfault": 1,
-                "pgpgin": 2426,
-                "pgpgout": 2199,
-                "rss": 843776,
-                "rss_huge": 0,
-                "total_active_anon": 421888,
-                "total_active_file": 36864,
-                "total_cache": 86016,
-                "total_dirty": 0,
-                "total_inactive_anon": 421888,
-                "total_inactive_file": 49152,
-                "total_mapped_file": 53248,
-                "total_pgfault": 1587,
-                "total_pgmajfault": 1,
-                "total_pgpgin": 2426,
-                "total_pgpgout": 2199,
-                "total_rss": 843776,
-                "total_rss_huge": 0,
-                "total_unevictable": 0,
-                "total_writeback": 0,
+                "active_anon": 270336,
+                "active_file": 135168,
+                "anon": 246484992,
+                "anon_thp": 4194304,
+                "file": 325484544,
+                "file_dirty": 0,
+                "file_mapped": 170582016,
+                "file_writeback": 0,
+                "inactive_anon": 250257408,
+                "inactive_file": 325619712,
+                "kernel_stack": 2703360,
+                "pgactivate": 62898,
+                "pgdeactivate": 0,
+                "pgfault": 2150971515,
+                "pglazyfree": 207999,
+                "pglazyfreed": 0,
+                "pgmajfault": 0,
+                "pgrefill": 0,
+                "pgscan": 0,
+                "pgsteal": 0,
+                "shmem": 0,
+                "slab": 8112800,
+                "slab_reclaimable": 5753632,
+                "slab_unreclaimable": 2359168,
+                "sock": 200704,
+                "thp_collapse_alloc": 0,
+                "thp_fault_alloc": 0,
                 "unevictable": 0,
-                "writeback": 0
+                "workingset_activate": 0,
+                "workingset_nodereclaim": 0,
+                "workingset_refault": 0
             },
             "usage": {
-                "max": 7860224,
-                "pct": 0.000672283359618831,
-                "total": 1409024
+                "max": 0,
```

Comment:
Does this 0 look right to you?

Reply:
This is a case where V1 and V2 might be different. `Max` Just indicates the cgroup's upper threshold, with `0` meaning there is no upper limit.